python/ray/worker.py

```diff
@@ -1381,7 +1416,7 @@ def _init(address_info=None,
           plasma_directory=None,
           huge_pages=False,
           include_webui=True,
-          use_raylet=False):
+          use_raylet=None):
```

Comment:
This was a "bug" which was preventing many of the xray tests from actually using the xray code path (which is why I'm also commenting out a bunch of the xray tests in this PR).